plugins/tableselection/plugin.js

```diff
@@ -16,6 +16,10 @@
 		insertRow,
 		insertColumn;
 
+	function _isWidget( element ) {
```

Comment:
Actually there isn't a need to prefix such function with `_` as it's not a method. `isWidget` would be enough.